docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml

```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
         package map. To examine a package map, select a package map from the
         list.</para>
 
-        <para>You can select the <emphasis role="bold">Validate Package
-        Map</emphasis> tab to validate a package map.</para>
+        <sect3>
+          <title id="Packagemap_Actions">Packagemap Actions</title>
 
-        <para><figure>
-            <title>Validate Package Maps</title>
+          <para>To update package mappings, you would edit the packagemap file
+          (or a copy) add and activate the new one, and then delete the old
+          one. You can do this through the interface in ECLWatch.</para>
 
-            <mediaobject>
-              <imageobject>
-                <imagedata fileref="../../images/ECLWaPkMgr.jpg" />
-              </imageobject>
-            </mediaobject>
-          </figure></para>
+          <para><emphasis role="bold">Package Map Add </emphasis></para>
+
+          <para><figure>
+              <title>Add Package Maps</title>
+
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="../../images/ECLWA470A.jpg"
+                             vendor="ECLWatchSS" />
+                </imageobject>
+              </mediaobject>
+            </figure>To Add a packagemap to the target cluster:</para>
+
+          <orderedlist>
+            <listitem>
+              <para>Select the package map to add by checking the box next to
+              it.</para>
+            </listitem>
+
+            <listitem>
+              <para>Press the <emphasis role="bold">Add</emphasis> action
+              button and open the Add Package Map dialog.</para>
+            </listitem>
+
+            <listitem>
+              <para>Press the <emphasis role="bold">Select Package
+              File</emphasis> button and select the package files to
+              add.</para>
+            </listitem>
+
+            <listitem>
+              <para>Choose the <emphasis role="bold">Target </emphasis>to
+              associate the package map with.</para>
+            </listitem>
+
+            <listitem>
+              <para>Choose a Process FIlter /*what is command line
+              equiv?*/</para>
```

Comment:
btw, I don't think I support process filter on command line.  It's one of those complex things that if I start to think people really want or need it I will add but over the last 5 years or so no one seems to be needing it.